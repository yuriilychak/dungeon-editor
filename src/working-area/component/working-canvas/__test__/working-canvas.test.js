import React from "react";

import {createMount} from "@material-ui/core/test-utils";

import WorkingCanvas from "../working-canvas";

describe("working-canvas test",()=> {
    it ( "working-canvas snapshot", () => {
        const wrapper = createMount()(
            <WorkingCanvas onGetCanvasRef={jest.fn()} />
        );
        expect(wrapper.html()).toMatchSnapshot();
    });
});
