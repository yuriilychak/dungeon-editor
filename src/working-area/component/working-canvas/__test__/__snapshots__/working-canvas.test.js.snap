// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`working-canvas test working-canvas snapshot 1`] = `"<canvas class=\\"working-canvas-root\\"></canvas>"`;
